deps/rabbitmq_aws/src/rabbitmq_aws.erl

```diff
@@ -543,10 +544,18 @@ ensure_credentials_valid() ->
 %% @end
 api_get_request(Service, Path) ->
   rabbit_log:debug("Invoking AWS request {Service: ~p; Path: ~p}...", [Service, Path]),
+  api_get_request_with_retries(Service, Path, ?LINEAR_BACK_OFF_MILLIS, ?MAX_RETRIES).
+
+api_get_request_with_retries(Service, Path, SleepTime, Retries) ->
   ensure_credentials_valid(),
   case get(Service, Path) of
     {ok, {_Headers, Payload}} -> rabbit_log:debug("AWS request: ~s~nResponse: ~p", [Path, Payload]),
-                                 {ok, Payload};
+      {ok, Payload};
     {error, {credentials, _}} -> {error, credentials};
-    {error, Message, _}       -> {error, Message}
+    {error, Message, _}       ->
+      case Retries of
+        0 -> {error, Message};
+        _ -> timer:sleep(SleepTime),
+             api_get_request_with_retries(Service, Path, SleepTime, Retries - 1)
```

Comment:
maybe put a log message here about what the error is, and how many retries left?

Reply:
Added